hooks/tfupdate.sh

```diff
@@ -48,12 +54,16 @@ function per_dir_hook_unique_part {
 #   args (string with array) arguments that configure wrapped tool behavior
 #######################################################################
 function run_hook_on_whole_repo {
-  local -r args="$1"
+  declare -a -r args=("$@")
+
+  local expand_args=()
```

Comment:
For consistency and precision:
```suggestion
  local -a expand_args=()
```